src/metabase/public_settings.clj

```diff
@@ -281,6 +281,17 @@
   :type       :string
   :default    "Metabase")
 
+(defsetting loading-message
```

Comment:
Should this be a EE-only thing? I though that settings needs to be defined elsewhere if they are EE-only. Not an issue here, but just noticed.

Reply:
Good point! Other EE-only settings are also there, let's ask @metabase/backend-developers

Reply:
seems like there's lots of defsettings in the enterprise backend:
```shell
❯ ag defsetting enterprise -c
enterprise/backend/src/metabase_enterprise/advanced_config/models/pulse_channel.clj:2
enterprise/backend/src/metabase_enterprise/embedding/utils.clj:2
enterprise/backend/src/metabase_enterprise/sso/integrations/sso_settings.clj:26
enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj:4
```

Makes sense to have this there

Reply:
Another thought. There are two strings that are overridable now. Do we know the approximate cardinality of this set? For two such strings, individual `defsetting`s seems fine. But if we are going to grow to 10 or 50 this will get onerous and not such a great fit.

Reply:
I'm on the same page as @dpsutton here. Another question is how do we expect this to work when i18n is in play? Seems like if you had a custom message here it would be in that specific language and we wouldn't be able to i18n it which could be a problem for people who have users with different User locales

Reply:
For the latter, this setting is not the actual message but only a key of possible messages defined here https://github.com/metabase/metabase/blob/0382372aad3f4893b91d39dc3cdca4ac73d3d648/enterprise/frontend/src/metabase-enterprise/whitelabel/lib/loading-message.ts#L3. So localization shouldn't be an issue.

Reply:
@ranquild @alxnddr let's merge this as is. We have an [issue](https://github.com/metabase/metabase/issues/23414) to clean up enterprise settings cohesively. Things like app colors, some email config, etc. are in the regular oss locations just like this one. We'll clean them up in a correct way rather than just ad-hoc.